polaris/dataset/_adapter_utils.py

```diff
@@ -0,0 +1,41 @@
+import zarr
+import biotite.structure as struc
+from biotite.structure import Atom
+
+
+# load nested zarr file to dictionary
+def load_group_as_numpy_arrays(group):
+    """Load zarr group as dictionary of numpy arrays"""
+    arrays = {}
+    for key, item in group.items():
+        if isinstance(item, zarr.core.Array):
+            arrays[key] = item[:]
+        elif isinstance(item, zarr.hierarchy.Group):
+            arrays[key] = load_group_as_numpy_arrays(item)
+    return arrays
+
+
+def zarr_to_pdb(group: zarr.Group):
```

Comment:
This func is required by `_adaptor`, it would end up with a cycle import if moved to `polaris/dataset/converters/_pdb.py`. 
How about rename the file to `_pdb_utils.py`?